src/containers/Permission.jsx

```diff
@@ -36,18 +53,41 @@ const Permission = ({ match }) => {
             Application: {appName.toUpperCase()}
           </Typography>
           <Typography variant="h2">Permission: {permissionName}</Typography>
-          <List>
-            {queryResult.data[0].attributes.permissions[permissionName]
-              .verbs ? (
-              queryResult.data[0].attributes.permissions[
-                permissionName
-              ].verbs.map(permission => (
-                <ListItemText key={permission.name}>{permission}</ListItemText>
-              ))
-            ) : (
-              <ListItemText>ALL</ListItemText>
-            )}
-          </List>
+          <Typography variant="h3">
+            {queryResultApps.data.length > 0
+              ? !queryResultApps.data[0].attributes.permissions[permissionName]
```

Comment:
for code readability, could we extract this variable `queryResultApps.data[0].attributes.permissions[permissionName]` used a lot?